autobuild/windows/autobuild_windowsinstaller_1_prepare.ps1

```diff
@@ -20,13 +22,26 @@ param(
 echo "Install Qt..."
 # Install Qt
 pip install aqtinstall
+if ( !$? )
+{
+		throw "pip install aqtinstall failed with $LastExitCode"
+}
+
 echo "Get Qt 64 bit..."
 # intermediate solution if the main server is down: append e.g. " -b https://mirrors.ocf.berkeley.edu/qt/" to the "aqt"-line below
 aqt install --outputdir C:\Qt 5.15.2 windows desktop win64_msvc2019_64
+if ( !$? )
+{
+		throw "64bit Qt installation failed with $LastExitCode"
+}
 
 echo "Get Qt 32 bit..."
 # intermediate solution if the main server is down: append e.g. " -b https://mirrors.ocf.berkeley.edu/qt/" to the "aqt"-line below
 aqt install --outputdir C:\Qt 5.15.2 windows desktop win32_msvc2019
+if ( !$? )
+{
+		throw "32bit Qt installation failed with $LastExitCode"
```

Comment:
```suggestion
		throw "32bit Qt installation failed with exit code $LastExitCode"
```

Reply:
This and all others addressed via `sed -re 's/failed with \$LastExitCode/failed with exit code $LastExitCode/' -i $(find -name '*.ps1')`.